custom/panel_templates/Default/integrations/minecraft/minecraft_servers.tpl

```diff
@@ -78,12 +78,23 @@
                                     <label for="inputDefaultServer">{$DEFAULT_SERVER}</label>
                                     <select id="inputDefaultServer" class="form-control" name="default_server">
                                         {if $SERVERS|count > 1}
-                                        <option value="none" {if $DEFAULT_SERVER_VALUE eq 0} selected{/if}>{$NO_DEFAULT_SERVER}</option>
+                                            <option value="none" {if $DEFAULT_SERVER_VALUE eq 0} selected{/if}>{$NO_DEFAULT_SERVER}</option>
                                         {/if}
                                         {if isset($SERVERS) && count($SERVERS)}
-                                        {foreach from=$SERVERS item=server}
-                                        <option value="{$server.id}" {if $server.is_default eq 1} selected{/if}>{$server.name}</option>
-                                        {/foreach}
+                                            {foreach from=$SERVERS item=server}
+                                                <option value="{$server.id}" {if $server.is_default eq 1} selected{/if}>{$server.name}</option>
+                                            {/foreach}
+                                        {/if}
+                                    </select>
+                                </div>
+
+                                <div class="form-group">
+                                    <label for="inputGroupSyncServer">{$GROUP_SYNC_SERVER}</label>
+                                    <select id="inputGroupSyncServer" class="form-control" name="group_sync_server">
```

Comment:
If there are no servers, this will be an empty select menu without any options. Perhaps have some message? Or no servers like the select menu above

Reply:
Weirdly the default server dropdown has no default either, will fix that now anyways...
<img width="1463" alt="Screen Shot 2022-08-11 at 21 09 47" src="https://user-images.githubusercontent.com/26070412/184277891-5a3770b9-8705-47c2-bfaa-a881e0d2bd40.png">

Thanks for the input